force-app/main/default/lwc/displayRelatedObjectsData/displayRelatedObjectsData.js

```diff
@@ -230,7 +230,7 @@ export default class DisplayRelatedObjectsData extends LightningElement {
 
         if ((records[i] + '').includes('servlet.shepherd/version/download/')) {
             cellData.value = records[i];
-            cellData.style = 'border: 1px solid black; text-align: center;';
+            cellData.style = 'border: 1px solid black; text-align: center; background-color:transparent;';
```

Comment:
Was this change tested with Sub Total and Total generation template also? i dont see this tag ebing added for those cells.